src/agentversion

```diff
@@ -1 +1 @@
-2.162.1
+2.164.0
```

Comment:
Any reason you're bumping to 164?

Reply:
2.163.1 is now 2.160.1 + 1 cherry pick change, I can make it 2.163.2 if think that is best

Reply:
Shouldn't we not be changing this on master at all?

Reply:
I agree with this completely and have some thoughts around this, but how it's been done so far is to keep updating the agentversion on master.

Reply:
Ok, that's fine. I don't think we should jump to 164 though - I think we want to keep rolling on 163 through next sprint to maintain consistency (staying even with mseng)